department-of-veterans-affairs/abd-vro
# Expose Special issue types endpoint throughout the BIP API service

The EE team has requested that the special_issue_types endpoint be exposed via a new rabbitmq queue on the bip exchange. This PR introduces many of the necessary components towards making that endpoint available for EE consumption. Accompanying the real bip data, is a mock implementation that can be used for local development.

## What was the problem?
special_issue_types were needed.

Associated tickets or Slack threads:
- #3213

## How does this fix it?[^1]
Adds the special_issue_types endpoint to all the necessary components of the bip service.

## How to test this PR
- submit a rabbitmq request to the newly create getSpecialIssueTypesQueue


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
